es6-classes/shape.js

```diff
@@ -0,0 +1,15 @@
+/* exported Shape */
+
+class Shape {
+  constructor(area, perimeter) {
+    this.area = area;
+    this.perimeter = perimeter;
+  }
+
+  describe() {
+    return 'This is the area and perimeter ' + this.area + ' ' + this.perimeter;
+  }
+}
+
+const shape = new Shape();
```

Comment:
You should be passing in 2 arguments here. One representing the `area` & the other `perimeter`